bokehjs/src/lib/core/util/color.ts

```diff
@@ -71,21 +71,25 @@ export function valid_rgb(value: string): boolean {
 
   // if '.' and then ',' found, we know decimals are used on rgb
   if (new RegExp(".*?(\\.).*(,)").test(value))
-    throw new Error(`color expects integers for rgb in rgb/rgba tuple, received ${value}`)
+    return false
+    // throw new Error(`color expects integers for rgb in rgb/rgba tuple, received ${value}`)
```

Comment:
what's going on here with keeping commented code?

Reply:
I view those as valuable, just the API needs a clean up, so that we have functions that simply answer a question and another that give feedback (e.g for usage in api/plotting). I will do in another PR.